crates/voicevox_core_c_api/src/lib.rs

```diff
@@ -661,7 +684,7 @@ pub unsafe extern "C" fn voicevox_synthesizer_tts(
 /// @param [in] json 解放する json データ
 ///
 /// # Safety
-/// @param voicevox_audio_query で確保されたポインタであり、かつ呼び出し側でバッファの変更を行われていないこと
+/// @param ライブラリ側で確保されたポインタであり、かつ呼び出し側でバッファの変更を行われていないこと
```

Comment:
ここは多分 #532 の範囲なので、外していいと思います。

```suggestion
/// @param voicevox_audio_query で確保されたポインタであり、かつ呼び出し側でバッファの変更を行われていないこと
```